CHANGES.md

```diff
@@ -7,6 +7,18 @@ Known issues:
 
 ## not yet released
 
+## 5.2.0
+
+- Add support for creating and managing NFS shared volumes. New `triton volume`
```

Comment:
Add note that `triton volume` will remain hidden until server-side changes are shipped.